packages/cli-platform-ios/src/tools/selectFromInteractiveMode.ts

```diff
@@ -20,13 +20,13 @@ export async function selectFromInteractiveMode({
   let newScheme = scheme;
   let newMode = mode;
 
-  if (projectInfo.schemes.length > 1) {
+  if (projectInfo && projectInfo.schemes.length > 1) {
```

Comment:
knowing that `projectInfo` can be undefined for workspaces (only `schemes` are returned from `xcodebuild -list` then, shall we adjust this to account for a workspace schemes? now it will automatically select `scheme` and `mode` passed to that fn